OnBoard-Media-Control/main.cpp

```diff
@@ -0,0 +1,19 @@
+#include "../song_player/SongPlayer.h"
+#include "OnBoardMediaControl.h"
+
+int main(int argc, char *argv[])
+{
+
+    QScopedPointer<OnBoardMediaControl> app;
+    app.reset(new OnBoardMediaControl(argc, argv));
+
+    SongPlayer player;
+    const QStringList arguments = QCoreApplication::arguments();
+    if (arguments.size() > 1)
+        player.playFile(arguments.at(1));
+
+    player.resize(300, 60);
+    player.show();
```

Comment:
None of this stuff should be here. All of it should be encapsulated in `OnBoardMediaControl`. Look at all the other projects and follow suit.